src/components/Header/Header.css

```diff
@@ -14,4 +14,23 @@
   top: 1%;
   left: 2%;
   transform: rotate(-5deg);
+}
+
+@media (max-width: 576px) { 
+  .header {
+    display: flex;
+    justify-content: space-around;
+    width: 100vw;
+    height: 10vh;
+  }
+
+  .title-logo {
+    position: static;
+    height: 100%;
+  }
+
+  .logo {
+    position: static;
+    height: 100%;
+  }
```

Comment:
The header needs some refactoring. The way it is styled using absolute positioning isn't great for responsive layouts. There isn't enough room on a phone screen to make something like this work well. I tried to set it to a phone-friendly layout the best. It isn't great and needs to be reworked.